source/guide_resilio.rst

```diff
@@ -0,0 +1,124 @@
+.. highlight:: console
+
+.. author:: Martin Porsch <https://martin.pors.ch/martin@pors.ch>
+
+.. sidebar:: About
+  
+  .. image:: _static/images/resilio.png 
+      :align: center
+
+############
+Resilio Sync
+############
+
+Resilio-Sync_ (formerly BitTorrent Sync) is a file syncing service similar to Dropbox that works with private peer to peer connections between connected devices.
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of 
+
+  * supervisord_
+  * domains_
+
+Prerequisites
+=============
+
+We need to set up some directories in order to make it work. ``~/bin/resilio`` is where we will put the executable and ``~/.sync`` is wehre Resilio-Sync wants to store all config files.
+
+::
+
+ [isabell@stardust ~]$ mkdir -p ~/bin/resilio
+ [isabell@stardust ~]$ mkdir ~/.sync
+ [isabell@stardust ~]$ 
+
+Furthermore, we need a free port that Resilio-Sync can listen to. To generate a currently unoccupied port run:
+
+::
+
+ [isabell@stardust ~]$ FREEPORT=$(( $RANDOM % 4535 + 61000 )); ss -ln src :$FREEPORT | grep $FREEPORT && echo "try again" || echo $FREEPORT
```

Comment:
Please use `.. include:: includes/generate-port.rst`

Reply:
Github wont display the include. I cannot verify if it worked.